scripts/apps/authoring/authoring/directives/ArticleEditDirective.ts

```diff
@@ -322,7 +327,8 @@ export function ArticleEditDirective(
                                     scope.articleEdit.$setDirty();
                                 }
                             } else {
-                                scope.save();
+                                // force refresh of editor3 fields
+                                scope.save().then(() => scope.refresh());
```

Comment:
Why are you calling `scope.refresh` here instead of after resetting editor3 fields? It would be clearer to have them together so it's obvious that they are handling the same thing.